CONTRIBUTING.md

```diff
@@ -84,23 +95,23 @@ yarn install
 git checkout -b my-topic-branch
 ```
 
-6. Make changes, commit and push to your fork:
+6. Make changes, commit, and push to your fork:
 
 ```bash
 git push -u origin HEAD
 ```
 
-7. Go to [the repository](https://github.com/mui/material-ui) and make a Pull Request.
+7. Go to [the repository](https://github.com/mui/material-ui) and open a pull request.
 
-The core team is monitoring for Pull Requests. We will review your Pull Request and either merge it, request changes to it, or close it with an explanation.
+The core team actively monitors for new pull requests.
+We will review your PR and either merge it, request changes to it, or close it with an explanation.
 
 ### Trying changes on the documentation site
 
-The documentation site is built with MUI and contains examples of all the components.
-This is the best place to experiment with your changes.
-It's the local development environment used by the maintainers.
+The documentation site is built with MUI and contains examples of all of the components.
```

Comment:
```suggestion
The documentation site is built with Material UI and contains examples of all of the components.
```

👀 🙈 🫣 (at least the way I'm reading it, it also sounds like this sentence is _just_ referring to Material UI)

Reply:
Good catch!